Userland/Libraries/LibJS/Runtime/Intl/SegmentIterator.cpp

```diff
@@ -19,7 +19,7 @@ NonnullGCPtr<SegmentIterator> SegmentIterator::create(Realm& realm, Segmenter& s
     // 4. Set iterator.[[IteratedString]] to string.
     // 5. Set iterator.[[IteratedStringNextSegmentCodeUnitIndex]] to 0.
     // 6. Return iterator.
-    return realm.heap().allocate<SegmentIterator>(realm, realm, segmenter, move(string), segments).release_allocated_value_but_fixme_should_propagate_errors();
+    return realm.heap().allocate<SegmentIterator>(realm, realm, segmenter, string, segments).release_allocated_value_but_fixme_should_propagate_errors();
```

Comment:
Similar to JsonArray, the one caller of this method can `move` its string in, so IMO that would be the better fix.